lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -399,11 +425,21 @@ def _is_small_course(course_key):
     return is_small_course
 
 
-def _section_student_admin(course, access):
+def _section_student_admin(course, access, unique_student_identifier, problem_to_reset):
```

Comment:
@dan-f I removed all three action links on ```staff_debug_info``` modal after talking to @explorerleslie and @cpennington and made a single link which will redirect the user to instructor dashboard and auto populate the username and problem id in section and we need to get the problem link back to courseware that's why I made this change to return all these values in ```section_data```.